inside.go

```diff
@@ -19,8 +19,12 @@ func (f *Interface) consumeInsidePacket(packet []byte, fwPacket *FirewallPacket,
 		return
 	}
 
-	// Ignore packets from self to self
+	// Immediately forward packets from self to self
 	if fwPacket.RemoteIP == f.myVpnIp {
+		_, err := f.readers[q].Write(packet)
```

Comment:
In group discussion, we think this should be limited to macos only and leave the old behavior for all other oss

Reply:
Wrote it up with a const, in the hopes that a const gives the compiler the best opportunity to optimize the code path as it sees fit. Happy to implement the o/s specific path other ways if desired.